gtda/images/filtrations.py

```diff
@@ -349,3 +350,135 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class DilationFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on distances to activated
+    pixels.
+
+    The dilation filtration assigns to each deactivated pixel of an image a
+    pixel value corresponding to the Manhattan distance between the pixel and
+    its closest activated pixel. The dilation process is iterative and the
+    number of iterations `n_iterations` sets the maximum distance considered.
+    Deactivated pixels further away from activated pixels are assigned the
+    maximum Manhattan distance between two pixels on the image and activated
+    pixels are assigned a value of 0.
+
+    Parameters
+    ----------
+    n_iterations : int or None, optional, default: ``None``
+        Number of iterations in the dilation process. ``None`` means dilation
```

Comment:
I guess I am saying that "dilation over the full image" seems like a little bit of jargon, and I for one do not really know what it means!

Reply:
> Dilation reaches all deactivated pixels?

Reply:
Your alternative is good!